README.md

```diff
@@ -1,5 +1,27 @@
 # nested-pandas
-WORK IN PROGRESS: Efficient Pandas representation for nested associated datasets.
+Efficient Pandas representation for nested associated datasets.
+
+Nested-Pandas extends the Pandas package with tooling and support for nested 
+dataframes packed into values of top-level dataframe columns. Pyarrow is used
+intrinsically to aid in scalability and performance.
+
+![image](./nestedframe.png)
+
+Nested-Pandas is motivated by time-domain Astronomy use-cases, where we see
```

Comment:
optional nit: s/use-cases/use cases

Is a bit more common than the hyphenated form, and we ware already using a lot of hyphenated words (just a personal aesthetic preference, so very much an optional nit)